src/release.rs

```diff
@@ -48,8 +48,7 @@ pub static RELEASE_TRIPLES: Lazy<BTreeMap<&'static str, TripleRelease>> = Lazy::
 
     // Windows.
 
-    // The -shared and -static parts of the triple are a lie. But the code
-    // handles it fine.
+    // The -shared part of the triple is a lie. But the code handles it fine.
```

Comment:
As a potential future followup we can strip the `-shared` part from the triple. The only reason it is there is to disambiguate from the static builds. Now since there are only dynamically linked builds, there's no need for the label.

I'll file an issue to track this.

Reply:
Cool, I can do that too.